func/omp/simple_for.cpp

```diff
@@ -0,0 +1,24 @@
+#include <omp.h>
+#include <stdio.h>
+#include <faasm/faasm.h>
+
+bool failed = false;
+#define ITERATIONS 1
+
+// Use base values else than 0 or 1 because they are often reset to that
+#define BASE_VALUE 100
+
+FAASM_MAIN_FUNC() {
+    int var = BASE_VALUE + ITERATIONS;
+
+    #pragma omp parallel for num_threads(1) default(none) shared(var)
+    for (int i = 0; i < ITERATIONS; i++) {
+        var--;
```

Comment:
Will this result in a thread-safe operation? I don't think the C++ decrement is guaranteed to be atomic in all scenarios, let alone when it's been compile to machine code via wasm

Reply:
No it's not, but this is a sequential loop (setting `num_threads(1)` creates only 1 thread in fork that the main thread waits on in the `fork` call